packages/itwinui-react/src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -76,4 +155,36 @@ export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
   );
 };
 
+export const FileUploaded = (props: {
+  /**
+   * Svg icon for uploaded file output.
+   */
+  icon: JSX.Element;
+  /**
+   * File label for uploaded file output.
+   */
+  label: string;
+  /**
+   * File description for uploaded file output.
+   */
+  description: string;
+  /**
+   * Action element for uploaded file.
+   */
+  action: JSX.Element;
```

Comment:
I think this prop could be optional. Maybe users don't want any action on uploaded file.